src/content/protocols/sky.md

```diff
@@ -0,0 +1,397 @@
+
+The Sky protocol relies on the provider Chronicle for price feeds of collateral assets. The Chronicol protocol uses a system of validators hosted by various third parties such as Bitcoin Suisse, ETHGlobal, Gitcoin, and Etherscan. Validators push price updates on-chain. We found that any changes to the validator set are subject to a 7 days exit window. We therefore assessed that Chronicle is a Stage 1 dependency and explained it in detail in a [dedicated report](./chronicle.md).
+
+An Oracle Security Module (`OSM`) enforces a 1-hour delay on price updates, and the Sky governance can freeze the current price to prevent further updates in case of emergency. The price feed provider could be changed through a governance proposal.
+
+> Autonomy score: High
+
+## Exit Window
+
+All permissions within Sky are held by the onchain Sky Governance. There are no external accounts or multisigs in control.
+
+Governance is controlled by `MKR`/`SKY` token holders who locked their tokens to earn voting power, which allows them to submit and vote on respective proposals, or delegate their votes to another user. Any token holder can create a proposal, and the protocol uses a system of continuous approval, explained in more detail in the [governance section](#exit-window-1).
+
+The minimum delay between approval and execution of a governance proposal is currently **18 hours**. Governance proposals have a recurring weekly and monthly schedule. Those proposals usually have a courtesy delay duration of 2-3 days before taking effect, but proposals remain possible at any time with the minimum delay.
+
+Emergency measures allow the governance to pause certain contracts through a governance proposal without being subject to the mandatory delay. In addition to that, an _Emergency Shutdown Module_ (ESM) exists and can irreversibly shutdown the entire protocol if **500'000** `MKR` tokens are sent to the Emergency Shutdown Contract. Funds sent to the contract are considered lost, no matter the emergency status. During an emergency shutdown, users can retrieve their supplied assets, but no additional supply or borrowing is allowed. This is explained further in the technical [governance section](#exit-window-1).
+
+> Exit Window score: High
+
+## Accessibility
+
+Sky has a main frontend at [sky.money](https://sky.money). The frontend is not self-hostable nor open source, but multiple other access points exist with Sky-specific apps such as [Spark](https://spark.fi) or third-party apps like [DeFiSaver](https://defisaver.com) or [SummerFi Pro](https://pro.summer.fi). These apps build an acceptable backup solution in case of failure of the official frontend.
+
+> Accessibility score: Low
+
+## Conclusion
+
+The Sky protocol exposes critical permissions that are not protected with an Exit Window of at least 7 days or a Security Council and thus earns a High centralization risk score for its Upgradeability and Exit Window dimensions.
```

Comment:
Exit Window -> _Exit Window_
Security Council -> _Security Council_

High centralization risk score for Upgradeability -> High _Upgradeability_ and _Exit Window_ scores